website/pages/partials/jwt_claim_mapping_details.mdx

```diff
@@ -0,0 +1,71 @@
+## Trusted Identity Attributes via Claim Mappings
+
+Data from JWT claims can be returned from the authentication step as trusted
+identity attributes for use in binding rule selectors and bind name
+interpolation.
+
+Control of which claims are mapped to which identity attributes is governed by
+the [`ClaimMappings`](#claimmappings) and
+[`ListClaimMappings`](#listclaimmappings). These are both maps of items to copy
+with elements of the form: `"<JWT claim>":"<attribute suffix>"`.
+
+The only difference between these two types of mappings is that `ClaimMappings`
+is used to map values which can be interpolated while `ListClaimMappings` is
+used to map to map lists of values that cannot be interpolated.
+
+Assume this is your config snippet:
+
+```json
+{ ...other fields...
+  "ClaimMappings": {
+    "givenName": "first_name",
+    "surname": "last_name"
+  },
+  "ListClaimMappings": {
+    "groups": "groups"
+  }
+}
+```
+
+This specifies that the values in the JWT claims `"givenName"` and `"surname"`
+should be copied to attributes named `"value.first_name"` and
+`"value.last_name"` respectively. Additionally the list of values in the JWT
+claim `"groups"` should be copied to an attribute named `"list.groups"`.
+
+Concretely:
+
+| Attributes         | Supported Selector Operations                      | Can be Interpolated |
+| ------------------ | -------------------------------------------------- | ------------------- |
+| `value.first_name` | Equal, Not Equal, In, Not In, Matches, Not Matches | yes                 |
+| `value.last_name`  | Equal, Not Equal, In, Not In, Matches, Not Matches | yes                 |
+| `list.groups`      | In, Not In, Is Empty, Is Not Empty                 | no                  |
+
+
+**TODO** If a claim is configured in claim_mappings, it must existing in the
+JWT or else the authentication will fail.
+
+### Claim Specifications and JSON Pointer
+
+The [`ClaimMappings`](#claimmappings) and
+[`ListClaimMappings`](#listclaimmappings) fields are used to point to data
+within the JWT. If the desired key is at the top of level of the JWT, the name
+can be provided directly. If it is nested at a lower level, a JSON Pointer may
+be used.
+
+Assume the following JSON data to be referenced:
```

Comment:
This "JSON data" is specifically the payload section of a JWT right? It might be worth being uber explicit and perhaps even including dummu `iss` and `aud` fields to reinforce what's going on here?

Reply:
Yeah this means the JWT claims.

Reply:
I'll extend vault's example here with some standard JWT claims and change the wording.